src/query/storages/index/src/bloom.rs

```diff
@@ -134,9 +140,11 @@ impl BlockFilter {
                 }
 
                 let filter = filter_builder.build()?;
+                if let Some(len) = filter.len() {
```

Comment:
does `filter.len()` return the NDV or just the number of values?

Reply:
the len is the number of distinct values.

Reply:
:ok_hand: